packages/lavapack/src/runtime.js

```diff
@@ -70,6 +70,17 @@
 })({ onStatsReady }) : () => {}
   const createKernel = // LavaMoat Prelude
 (function () {
+  const globalRef = globalThis;
+
+  const {
+    includeSnow,
+  } = {"includeSnow":false}
+
```

Comment:
```suggestion
  const includeSnow = false;
```
*(for clarity. in case this is to be kept in the first place for now, which is brought into question below)*